### ISSUE TITLE
Does LGB support dynamic learning rate? 

### ISSUE THREAD

**Issue by fydlzr** [microsoft/LightGBM#3546]
Does LGB support dynamic learning rate?  
Can lr be smaller and smaller during the training process?

And I just updated lgb from 2.3.1 to 3.0.0, does the old param "max_position" equal to "lambdarank_truncation_level"?

**Comment by StrikerRUS** [comment]
> Does LGB support dynamic learning rate?

Yes, it does. 
> learning_rates (list, callable or None, optional (default=None)) – List of learning rates for each boosting round or a customized function that calculates learning_rate in terms of current number of round (e.g. yields learning rate decay).
https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.train.html#lightgbm.train

Yes, formerly this param was named `max_position`. Please refer to https://github.com/microsoft/LightGBM/pull/2801/files.

**Comment by ljczxy** [comment]
我最近也遇到了做过问题，太感谢了

**Comment by ljczxy** [comment]
我最近也遇到了这个问题，感谢
